core/src/main/java/rdx/works/core/preferences/PreferencesManager.kt

```diff
@@ -64,15 +65,25 @@ class PreferencesManager @Inject constructor(
         }
     }
 
-    suspend fun markFirstPersonaCreated() {
+    suspend fun markFactorSourceBackedUp(id: String) {
         dataStore.edit { preferences ->
-            preferences[KEY_FIRST_PERSONA_CREATED] = true
+            val current = preferences[KEY_BACKED_UP_FACTOR_SOURCE_IDS]
+            if (current == null) {
+                preferences[KEY_BACKED_UP_FACTOR_SOURCE_IDS] = id
+            } else {
+                preferences[KEY_BACKED_UP_FACTOR_SOURCE_IDS] = listOf(current, id).joinToString(",")
+            }
         }
     }
 
-    suspend fun markImportFromOlympiaComplete() {
+    val isRadixBannerVisible: Flow<Boolean> = dataStore.data
+        .map { preferences ->
+            preferences[KEY_RADIX_BANNER_VISIBLE] ?: false
+        }
+
+    suspend fun changeRadixBannerVisibility(isVisible: Boolean) {
```

Comment:
I know it's too picky but I think it's nice to be aligned as much as possible with the naming of the native functions of Android/Kotlin, therefore `setRadixBannerVisibility` or `enableRadixBanner` or something more familiar, you get the point?